std/algorithm/comparison.d

```diff
@@ -17,6 +17,9 @@ $(T2 clamp,
 $(T2 cmp,
         $(D cmp("abc", "abcd")) is $(D -1), $(D cmp("abc", "aba")) is $(D 1),
         and $(D cmp("abc", "abc")) is $(D 0).)
+$(T2 either,
+        Return first parameter whose $(D bool)-conversion is $(D true), e.g.
+        $(D either(0, 42, 43)) returns $(D 42).)
```

Comment:
That wouldn't be the bool conversion. Should be something like: "Return first parameter `pk` that passes an `if (pk)` test, e.g. $(D either(0, 42, 43)) returns $(D 42)."